docs/connect.html

```diff
@@ -129,9 +129,11 @@ <h4><a id="connect_transforms" href="#connect_transforms">Transformations</a></h
     <p>The file source connector reads each line as a String. We will wrap each line in a Map and then add a second field to identify the origin of the event. To do this, we use two transformations:</p>
     <ul>
         <li><b>HoistField</b> to place the input line inside a Map</li>
-        <li><b>InsertField</b> to add the static field. In this example we'll indicate that the record came from a file connector</li>
+        <li><b>InsertField</b> to add the static field</li>
     </ul>
 
+    <p>In this example we'll indicate that the record came from a file connector.</p>
```

Comment:
fix the wrong format. 
before:
![image](https://user-images.githubusercontent.com/43372967/82002047-504dff00-968f-11ea-9332-9a28bd3fc1bb.png)
after:
![image](https://user-images.githubusercontent.com/43372967/82002116-85f2e800-968f-11ea-8d5f-4de48b5ddef1.png)